src/main/java/screens/GraphScene.java

```diff
@@ -19,32 +19,59 @@
  */
  /*package*/ class GraphScene extends Pane {
 
+    /**
+     * State for handling center queries.
+     */
     private final INodeHandler center;
+
+    /**
+     * State for handling information display queries.
+     */
     private final INodeHandler info;
+
+    /**
+     * The current state for node events.
+     */
     private INodeHandler state;
-     /**
+
+    /**
      * Event handler for when a node or edge is clicked.
      */
      private EventHandler<MouseEvent> click = event -> {
 
         if (event.getSource() instanceof DrawNode) {
+            /**
+             * DrawNode object that is linked to the Node object.
+             */
             DrawNode rect = (DrawNode) (event.getSource());
             state.handle(rect);
         } else if (event.getSource() instanceof Line) {
+            /**
+             * Line object.
```

Comment:
A new object/class (event) is being defined here with new variables not defined earlier. This is probably the reason checkstyle returns an error.